alvr/server/src/connection.rs

```diff
@@ -748,19 +778,69 @@ fn try_connect(mut client_ips: HashMap<IpAddr, String>) -> ConResult {
                 if let Some(stats) = &mut *STATISTICS_MANAGER.lock() {
                     stats.report_tracking_received(tracking.target_timestamp);
 
+                    // Handle hand gestures
+                    if let Switch::Enabled(controllers_config) = &settings.headset.controllers {
+                        if let Switch::Enabled(gestures_config) =
+                            &controllers_config.hand_tracking.use_gestures
+                        {
+                            let mut hand_gesture_manager_lock = hand_gesture_manager.lock();
+
+                            if tracking.hand_skeletons[0].is_some() {
+                                trigger_hand_gesture_actions(
+                                    gestures_button_mapping_manager.as_mut().unwrap(),
```

Comment:
Avoid using `.unwrap()` at all cost. a failed unwrap will mean a partial or complete crash of the server. unwrap the value using if-let.